packages/netlify-cms-core/src/actions/__tests__/entries.spec.js

```diff
@@ -78,5 +78,18 @@ describe('entries', () => {
       ]);
       expect(createEmptyDraftData(fields)).toEqual({});
     });
+
+    it('should not return an empty object for boolean widgets without a default value', () => {
+      const fields = fromJS([
+        {
+          name: 'post',
+          widget: 'boolean',
+          fields: [{ name: 'title', widget: 'boolean' }],

```

Comment:
`boolean` widgets don't have `fields` attrs, test for `{ name: 'boolean', widget: 'boolean' }` for example, which should output  `{ boolean: false }`

Reply:
I switched the parent back to object, like all the other tests. That was a leftover from some back and forth testing...